### ISSUE TITLE
TFT35 Blank Screen and non Responsive

### ISSUE THREAD

**Issue by pedrax202** [bigtreetech/BTT-TFT35-E3-V3.0#26]
Very similar to another ticket here I have recently added an SKR mini E3 v2.0 board and TFT35 V3 to my Ender 3 Pro. Both were working perfectly for about 2 or 3 days then during the middle of a print the print stopped and the TFT when blank. Rebooting didn't help, I tried to reflash firmware and it wouldn't take.
Power LED was lit on back of board and it would recognise an SD Card (green LED). The printer would fail to initialise as Octoprint could not see it. As soon as I disconnected board and installed the original Creality screen the printer boots fine and operates normally.

TFT was connected by EXP3 and RS232 to the E3 V2 board by TFT and EXP connection. Could not reset TFT and no other attempts to restore have worked.

**Comment by oldman4U** [comment]
Have you tried to remove the TFT from the metal plate it is mounted on?

**Comment by pedrax202** [comment]
Yes. Tried with it removed aswell. No improvement

**Comment by oldman4U** [comment]
Could u send a picture which shows the screen when it is white. Have you tried with only the serial connected  Not sure it works with only the exp connected but i can try.

**Comment by oldman4U** [comment]
OK, so I tested my screen with only the serial and only the EXP connected and in both cases the display turns on and shows the user interface (touchscreen).

Seems to be a broken mainboard, cable or TFT...;-(

Please let me know

**Comment by pedrax202** [comment]
> 
> 
> OK, so I tested my screen with only the serial and only the EXP connected and in both cases the display turns on and shows the user interface (touchscreen).
> 
> Seems to be a broken mainboard, cable or TFT...;-(
> 
> Please let me know

Hi Oldman4U. Tried each of the combinations and my screen stays off. Doesnt even illuminate. With the 10 pin cable in EXP3 the Power LED comes on but nothing happens. The printer will also not boot when the TFT is connected. I have contacted BTT through Aliexpress to try and get a replacement.

**Comment by oldman4U** [comment]
Makes sense. Please close this ticket as the problem seems to be hardware
related.

Thank you

pedrax202 <notifications@github.com> schrieb am So. 9. Aug. 2020 um 17:14:

**Comment by AlexKBR2** [comment]
Hey guys, I was so disheartened by this crappy problem, that I connected a wire to the 5v of one of the exp slots on the back of the tft35 and it connected randomly to the other pins, and, it came back to life! But it only works if I turn on EXP1 and EXP3... I don't know why. In this case, I used my old RAMPS 1.4. Even so, it is not behaving normally... The Display resets itself, and in the beginning the functions were all messed up, with the icons not corresponding to the functions, then I updated the firmware again and they returned to their normal positions. The two images below...
![WhatsApp Image 2023-07-22 at 21 28 56](https://github.com/bigtreetech/BTT-TFT35-E3-V3.0/assets/132751911/4060ddd0-7f99-4397-8ede-006d87a296d1)
![WhatsApp Image 2023-07-22 at 21 28 57](https://github.com/bigtreetech/BTT-TFT35-E3-V3.0/assets/132751911/9c33ae8f-ee1e-463d-bafc-34b5c016396a)
